packages/terra-clinical-result/src/ClinicalResultBloodPressure.jsx

```diff
@@ -176,44 +176,38 @@ const ClinicalResultBloodPressure = (props) => {
     const conceptDisplayElement = createConcpetDisplays(compareConceptDisplays);
 
```

Comment:
can we move the array declaration from line 138 to right before it's used?
```suggestion
const decoratedResultDisplay = []
```

Reply:
I made this update real quick.